test/CMakeLists.txt

```diff
@@ -124,6 +125,12 @@ if(UMF_BUILD_OS_MEMORY_PROVIDER AND LINUX) # OS-specific functions are implement
                  LIBS umf_utils numa)
 endif()
 
+if(UMF_BUILD_LIBUMF_PROVIDER_LEVEL_ZERO AND LINUX) # TODO
```

Comment:
Why TODO? Is there anything missing to support it on windows?

Reply:
At the moment we don't have the internal infrastructure to test it. I will try to include Windows ASAP in separate PRs